src/components/Comments/ContainerComment.js

```diff
@@ -182,7 +182,7 @@ export const ReplyComment = ({
   );
 };
 export const ContainerReply = ({children, isGrandchild}) => (
-  <View style={[{borderColor: isGrandchild ? '#fff' : colors.gray1}]}>{children}</View>
+  <View style={[{borderColor: isGrandchild ? COLORS.white : COLORS.gray9}]}>{children}</View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor property in the style object has been updated to use `COLORS.white` and `COLORS.gray9` instead of hard-coded color values. This change improves maintainability by centralizing color definitions, making it easier to update colors consistently across the application in the future.
```diff
- <View style={[{borderColor: isGrandchild ? '#fff' : colors.gray1}]}>{children}</View>
+ <View style={[{borderColor: isGrandchild ? COLORS.white : COLORS.gray9}]}>{children}</View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->